Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -140,7 +140,7 @@
   "loc.messages.ScriptStatusTimeout": "Unable to fetch script status due to timeout.",
   "loc.messages.PollingForFileTimeOut": "Unable to fetch script status due to timeout. You can increase the timeout limit by setting 'appservicedeploy.retrytimeout' variable to number of minutes required.",
   "loc.messages.InvalidPollOption": "Invalid polling option provided: %s.",
-  "loc.messages.MissingWebConfigParameters": "Some web.config parameters are missing",
+  "loc.messages.MissingAppTypeWebConfigParameters": "-appType attribute is missing in Web.config parameters.",
```

Comment:
Is apptype the only config param that can be missing? Should we also report if other required parameters are not present?